pagopa/io-app
# chore(IT Wallet): [SIW-1011] Add test toggle in user profile screen with playground entry

## Short description
This PR adds itwallet test toggle in user profile screen with itwallet playground entry item.

https://github.com/pagopa/io-app/assets/49342144/fcc19f4d-ff4d-492f-b147-56b9e31d03f2

## List of changes proposed in this pull request
- Add new persisted preference for itwallet flag
- Update user profile screen with new toggle and an entry point in playground section

## How to test
With io-dev-api-server go to user profile screen and activate `IT Wallet` in test environment section. Check with reacttotron if `isItWalletTestEnabled` is updated under `persistedPreferences` section. Check also if in the Playground section under user profile screen the new IT Wallet item appears.
